controller_manager:
  ros__parameters:
    update_rate: 100

    mani_arm_controller:
      type: joint_trajectory_controller/JointTrajectoryController
    end_effector_controller_1:
      type: position_controllers/GripperActionController
    end_effector_controller_2:
      type: position_controllers/GripperActionController
    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster

mani_arm_controller:
  ros__parameters:
    joints:
      - Revolute_1
      - Revolute_2
      - Revolute_3
      - Revolute_4
      - Revolute_5
      - Revolute_6
    command_interfaces:
      - position
    state_interfaces:
      - position
      - velocity

end_effector_controller_1:
  ros__parameters:
    joint: Slider_1

end_effector_controller_2:
  ros__parameters:
    joint: Slider_2